### PR TITLE
Update prevent-end-user-interaction-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Update the Png file to reflect what happens when it is enabled/disabled.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5699

File: microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ In Windows 10, versions 1703, hiding the interface will hide Microsoft Defender
 
 With the setting set to **Enabled**:
 
-:::image type="content" source="../../media/wdav-headless-mode-1703.png" alt-text="Windows Security without the shield icon and virus and threat protection section":::
+:::image type="content" source="../../media/wdav-headless-mode-off-1703.png" alt-text="Windows Security without the shield icon and virus and threat protection section":::
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md]:
```suggestion
:::image type="content" source="../../media/wdav-headless-mode-off-1703.png" alt-text="Screenshot of Windows Security without the shield icon and virus and threat protection sections":::
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**